<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecogAreaDialog</class>
 <widget class="QDialog" name="RecogAreaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RecogAreaDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBtnBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>9</y>
     <width>195</width>
     <height>237</height>
    </rect>
   </property>
   <property name="title">
    <string>R面</string>
   </property>
   <widget class="QPushButton" name="btn_block2_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block2_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block2_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block2_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block2_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block2_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block2_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block2_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block2_9">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBtnBox_1">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>195</width>
     <height>237</height>
    </rect>
   </property>
   <property name="title">
    <string>F面</string>
   </property>
   <widget class="QPushButton" name="btn_block1_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block1_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block1_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block1_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block1_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block1_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block1_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block1_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_block1_9">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
